From b67d87ae24e93581a014c894b35c7bcb80eb3c9a Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Mon, 14 Oct 2024 07:09:57 +0300
Subject: [PATCH] [SW-205285] gaudi3: defined a macro of d1 arc1 iccm full addr

Defined a macro of DIE1 ARC1 ICCM full address to support the
compilation for relocating MGMT code to DIE1 ARC1 ICCM.

Change-Id: I29725bc3d5b7da1913115759c04dc1035806ef05
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 60cac08..9629f07 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -50,6 +50,8 @@
 #define ARC2_DCCM_FULL_BASE		0x300007FD30E0000ull
 #define ARC2_DCCM_SIZE			SZ_32K
 
+#define D1_ARC1_ICCM_FULL_BASE		0x300007FDB040000ull
+
 #define ARC1_OFFLOAD_DCCM_FULL_BASE    0x0300007FDB0E0000ull
 #define ARC1_OFFLOAD_DCCM_SIZE         SZ_32K
 
